supplementary_style_guide/style_guidelines/mananged-services.adoc

```diff
@@ -0,0 +1,107 @@
+[[cloud-services]]
+= Cloud services
+
+Cloud services, also referred to as cloud services, are applications that run on a platform backed by data storage infrastructure and are hosted and managed by Red Hat on behalf of the customer. Managed cloud services provide rapid iteration and can more quickly deliver updated product functionality.
+
+This means a product is no longer running exclusively in the customer’s infrastructure but is a service running in the cloud that is operated and maintained by Red Hat.
+
+Red Hat is now responsible for ensuring the service is secure, scalable, performant, and available at all times.
+
+Cloud services documentation requires a shift from how documentation is typically created for an on-premise product. There is more emphasis on the content delivered within the user interface itself, a need to deliver content rapidly without necessarily having a set release cadence, and to adapt content to the needs of cloud service users. For example, there might not be a need for upgrade and migration content that’s typical for on-premise products.
+
+This section provides guidelines for creating content for the user interface, emphasizing accessibility. with the intent to provide consistency in how users understand and navigate across different cloud services.
+
+[[accessibility]]
+== Accessibility
+
+Ensure product documentation meets Section 508 compliance for accessibility. Accessibility standards are derived from the link:https://www.w3.org/TR/WCAG20/[Web Content Accessibility Guidelines (WCAGA)] and the link:https://www.access-board.gov/ict/[ICT Accessibility 508 Standards and 255 Guidelines].
```

Comment:
* Ensure -> Ensure that
* So the lead in sentence above (line 12) mentions that it's guidelines for creating content for the UI. But then here it talks about product docs. Should the above (line 12) specifically mention that it's product documentation _within_ the UI? (as opposed to button/help text, if this is true)

Reply:
Updated